### DISCUSSION TITLE
[General] Printer inventory

### DISCUSSION THREAD

**Discussion by FrenchHope** [glpi-project/glpi-agent#466] [General]
### Your question

Although all printers are present in local json inventory, not all are present in the computer inventory in the admin interface (maybe not even in database too, I don't know the table).

Is it possible to add all printers installed on a computer to computer inventory (printer name, printer driver, printer port).

**Comment by g-bougard** [comment]
Hi @FrenchHope 

not sure to understand your concern, what kind of printers are you talking about ?

Maybe some information is missing involving the printer is not imported in GLPI.

**Reply by FrenchHope** [reply]
Sorry for being imprecise, but I'd like to see printers that aren't directly connected (such as USB) inventoried as well: I'm thinking in particular of network printers to which you print via the IP protocol. I don't necessarily want to activate SNMP, I'd just like to find a reference to the installation of such printers as found in OCS inventory NG. A bit like software inventory

**Reply by FrenchHope** [reply]
On further investigation, I understand that these printers are indeed inventoried by the agent (there are traces of this in the XML), but I can't find any trace of them in the database on the server.
